### PR TITLE
Rest of mobile, roadmap JS, modal content (WIP)

### PR DESCRIPTION
**Pull request by BartoszBilejczyk**

### CODE REVIEW

File: js/countdown.js [hunk 1]
```diff
@@ -50,6 +53,14 @@ $(function() {
         }
 
       }, 1000);
+    },
+    progressbar: function() {
+      this.date = new Date();
+      var timeLeft = parseInt((this.endDate - this.date));
+      var icoTime = parseInt((this.endDate - this.startDate));
```
Comment (KonradFak) [inline js/countdown.js]: I think you don't need two brackets.

File: js/main.js [hunk 2]
```diff
@@ -11,4 +11,107 @@ $(window).ready(function() {
...
+
+  // roadmap active points
+  var roadmapRoadActive = $('.roadmap__road--active'),
+      now = new Date(),
+      icoStartDate = new Date('Wed, 07 Jun 2017 12:00:00 UTC'),
+      icoEndDate = new Date('Fri, 12 Jan 2018 12:00:00 UTC'),
+      adsharesDemoDate = new Date('Wed, 31 Jan 2018 12:00:00 UTC'),
+      listingOnExchangesDate = new Date('Wed, 28 Feb 2018 12:00:00 UTC'),
+      adUserServicesDate = new Date('Sat, 31 Mar 2018 12:00:00 UTC'),
+      gatewaysDate = new Date('Sat, 30 Jun 2018 12:00:00 UTC'),
```
Comment (KonradFak) [inline js/main.js]: This dates are pretty important variables. What do you think about declaring them as an object at the beggining of file:
```
{
   now: new Date(),
   icoStartDate: new Date('Wed, 07 Jun 2017 12:00:00 UTC'),
   ...
}
```
Then you could use function to create Array from them: `Object.keys(datesObj).map(function (key) { return dates.Obj[key] })`

File: js/main.js [hunk 3]
```diff
@@ -11,4 +11,107 @@ $(window).ready(function() {
...
+    for (i = roadmapStops.length; i > 0; i--) {
+      roadmapRoadActive.css(direction, margin * (2 * i - 1) + '%');
+
+      if (dates[0] > dates[i]) {
+        for (j = i; j > 0; j--) {
+          var elIndex = '0' + j;
+          var el = '.roadmap__road-stop[data-number=' + elIndex + ']';
+          $(el).addClass(active);
+        }
+        break;
```
Comment (KonradFak) [inline js/main.js]: Do you need this `break`?

File: js/countdown.js (hunk 1 above)
Comment (BartoszBilejczyk) [reply js/countdown.js]: I don't

File: js/main.js (hunk 3 above)
Comment (BartoszBilejczyk) [reply js/main.js]: I do

File: js/main.js [hunk 4]
```diff
@@ -1,4 +1,14 @@
 $(window).ready(function() {
+  datesObj = {
+    now: new Date(),
+    icoStartDate: new Date('Wed, 07 Jun 2017 12:00:00 UTC'),
+    icoEndDate: new Date('Fri, 12 Jan 2018 12:00:00 UTC'),
+    adsharesDemoDate: new Date('Wed, 31 Jan 2018 12:00:00 UTC'),
+    listingOnExchangesDate: new Date('Wed, 28 Feb 2018 12:00:00 UTC'),
+    adUserServicesDate: new Date('Sat, 31 Mar 2018 12:00:00 UTC'),
+    gatewaysDate: new Date('Sat, 30 Jun 2018 12:00:00 UTC')
+  }
```
Comment (KonradFak) [inline js/main.js]: If you don't use `var` keyword while declaring variables is assigned to `window` object and it is bad practise to add unnecessary things there, also missing semicolon ;)

File: js/main.js [hunk 5]
```diff
@@ -11,4 +21,90 @@ $(window).ready(function() {
...
+      }
+    }
+  }
+
+  var roadmapRoadActive = $('.roadmap__road--active'),
+      roadmapStops = $('.roadmap__road-stop'),
+      margin = 8.3333,
+      active = 'roadmap__road-stop--active',
+      roadmapBreakpoint = 1000;
+      dates = Object.keys(datesObj).map(function (key) { return datesObj[key] })
```
Comment (KonradFak) [inline js/main.js]: Here you also added `dates` to `window`, you can simply move semicolon after `dates` variable.

**Pull request merged**